Description: Disable simplecov
 Disable simplecov while running tests.
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/spec/spec_helper.rb
+++ b/spec/spec_helper.rb
@@ -1,15 +1,4 @@
 require 'rspec'
-require 'simplecov'
-
-if ENV['CI'] || (defined?(:RUBY_ENGINE) && RUBY_ENGINE != 'rbx')
-  begin
-    require 'codeclimate-test-reporter'
-    CodeClimate::TestReporter.start
-  rescue LoadError
-  end
-  SimpleCov.start
-end
-
 require 'omniauth-multipassword'
 
 Dir[File.expand_path('spec/support/**/*.rb')].each {|f| require f }
